### PR TITLE
openshift/ suggestions

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: openshift/deploy.sh [hunk 1]
```diff
@@ -74,14 +74,14 @@ function generate_and_deploy_config() {
...
-    -p FLASK_APP_SECRET_KEY=`echo -n $FLASK_APP_SECRET_KEY | base64` \
-    -p RDS_ENDPOINT="`echo -n $RDS_ENDPOINT | base64`" \
-    -p RDS_PASSWORD=`echo -n $RDS_PASSWORD | base64` \
+    oc process -f secrets-template.yaml -p AWS_ACCESS_KEY_ID=$(echo -n $AWS_ACCESS_KEY_ID | base64) \
+    -p AWS_SECRET_ACCESS_KEY=$(echo -n $AWS_SECRET_ACCESS_KEY | base64) \
+    -p GITHUB_API_TOKENS=$(echo -n $GITHUB_API_TOKENS | base64) \
+    -p GITHUB_OAUTH_CONSUMER_KEY=$(echo -n $GITHUB_OAUTH_CONSUMER_KEY | base64) \
+    -p GITHUB_OAUTH_CONSUMER_SECRET=$(echo -n $GITHUB_OAUTH_CONSUMER_SECRET | base64) \
+    -p FLASK_APP_SECRET_KEY=$(echo -n $FLASK_APP_SECRET_KEY | base64) \
+    -p RDS_ENDPOINT="$(echo -n $RDS_ENDPOINT | base64)" \
```
Comment (jpopelka) [inline openshift/deploy.sh]: I think the quotes can be removed here, WDYT ?

**Comment by jpopelka** [comment]
[test]

File: openshift/deploy.sh (hunk 1 above)
Comment (humaton) [reply openshift/deploy.sh]: lets do it.

**Comment by jpopelka** [comment]
amended, PTAL

**Pull request merged**